### PR TITLE
VS-1047 remove --location=US in bq command calls

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 1]
```diff
@@ -622,11 +622,11 @@ task BigQueryLoadJson {
 
         if [ $BQ_SHOW_RC -ne 0 ]; then
             echo "Creating a pre-vat table ~{dataset_name}.~{genes_table}"
-            bq --apilog=false --location=US mk --expiration=$DATE --project_id=~{project_id}  ~{dataset_name}.~{genes_table} ~{genes_schema}
+            bq --apilog=false mk --expiration=$DATE --project_id=~{project_id}  ~{dataset_name}.~{genes_table} ~{genes_schema}
         fi
 
         echo "Loading data into a pre-vat table ~{dataset_name}.~{genes_table}"
-        bq --apilog=false --location=US load  --project_id=~{project_id} --source_format=NEWLINE_DELIMITED_JSON  ~{dataset_name}.~{genes_table} ~{genes_path}
+        bq --apilog=false oad  --project_id=~{project_id} --source_format=NEWLINE_DELIMITED_JSON  ~{dataset_name}.~{genes_table} ~{genes_path}
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:629]:
```suggestion
        bq --apilog=false load --project_id=~{project_id} --source_format=NEWLINE_DELIMITED_JSON  ~{dataset_name}.~{genes_table} ~{genes_path}
```

**Review by mcovarr — COMMENTED** [review]
👁️

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl (hunk 1 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl]: Oops - thanks

**Comment by gbggrant** [comment]
Passing run of GvsCreateVATFromVDS [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%2010K%20Callset/job_history/7bb1410b-123c-4150-8a6b-f3d36234527a)
Passing run of GvsCallsetStatistics [here](https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20WGS%2010K%20Callset/job_history/ecc67d79-ecbf-41c6-bbee-52be83327d64)
(both are on the AoU 10K - so need your PMI-OPS account to see.

**Review by rsasch — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**